lib/rubocop/formatter/disabled_config_formatter.rb

```diff
@@ -140,23 +140,28 @@ def default_config(cop_name)
       end
 
       def output_cop_config(output_buffer, cfg, cop_name)
+        offending_files = @files_with_offenses[cop_name].uniq.sort
+        # Only modify the maximum if the files are not excluded one by one.
+        if offending_files.count > @exclude_limit && cfg[:exclude_limit]
```

Comment:
Use `size` instead of `count`. `count` should only be used when you have a block.